etc/profile-a-l/git-cola.profile

```diff
@@ -49,18 +62,20 @@ seccomp
 shell none
 tracelog
 
-# private-bin atom,bash,colordiff,emacs,fldiff,geany,gedit,git,git gui,git-cola,git-dag,gitk,gpg,gvim,leafpad,meld,mousepad,nano,notepadqq,python*,sh,ssh,vim,vimdiff,which,xed
+private-bin atom,basename,bash,colordiff,diff,emacs,envsubst,fldiff,geany,gedit,gettext,git,git gui,git-cola,git-dag,gitk,gpg,gpg-agent,gvim,leafpad,meld,mousepad,nano,notepadqq,nvim,pinentry,pinentry-gtk-2,ps,python*,sh,ssh,ssh-agent,tclsh,tr,vim,vimdiff,wc,which,xed
```

Comment:
",git gui," should the be "git-gui"?

There are also other pinentry binaries.

All the editors blow this up. Proposal:

```
private-bin basename,bash,colordiff,diff,…
# Replace with your editor.
private-bin atom,emacs,geany,gedit,…
```

Reply:
Is opposite for me ? https://imgur.com/SUmk6Ve, I'll change the editor and pinentry ones, I had only those two+-emacs,-curses...

Reply:
1. Then is the binary still `git` and not `git gui` (or do you have a `/usr/bin/git gui`).
2. I have
```
/usr/bin/pinentry
/usr/bin/pinentry-curses
/usr/bin/pinentry-gnome3
/usr/bin/pinentry-gtk -> pinentry-gtk-2
/usr/bin/pinentry-gtk-2
```

Reply:
1.> I have neither in `/bin/`?
2.> Yeah same for me, I choose gtk-2 to cache...